app/models/exercise.rb

```diff
@@ -47,7 +47,6 @@ def prerequisite_exercises
 
   memoize
   def git
-    # TODO: Change to sha, not HEAD
-    Git::Exercise.new(track.slug, slug, "HEAD", git_type)
+    Git::Exercise.new(track.slug, slug, git_sha, git_type)
```

Comment:
FYI, the reason this is currently like this is so that we always show the HEAD to maintainers, as I've not added in the ability for them to update their solutions yet. We might want to keep it that way for now?

Reply:
Sure!